app/controllers/papers_controller.rb

```diff
@@ -1,76 +1,79 @@
 class PapersController < ApplicationController
-  before_action :set_paper, only: [:show, :edit, :update, :destroy]
-  before_action :ensure_current_user, except: [:index, :show]
+	before_action :set_paper, only: [:show, :edit, :update, :destroy]
```

Comment:
You can also explicitly convert to spaces with ctrl-shift-p "convert tabs
to spaces" or something like that

On Thu, Sep 8, 2016 at 12:58 AM, wsf246 <notifications@github.com> wrote:

> In app/controllers/papers_controller.rb
> <https://github.com/briankung/projectcredo/pull/70#discussion_r77949808>:
>
> > @@ -1,76 +1,79 @@
> >  class PapersController < ApplicationController
> > -  before_action :set_paper, only: [:show, :edit, :update, :destroy]
> > -  before_action :ensure_current_user, except: [:index, :show]
> > +	before_action :set_paper, only: [:show, :edit, :update, :destroy]
>
> I do have those settings so I'm not sure why it's doing that.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/70/files/815df3663df0c5df7f52df03aca34b46bee0e2ad#r77949808>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtGxyEO2DuvCVJPNKzaIf8-FTCoQasXks5qn6P0gaJpZM4J2i6I>
> .
>



-- 
Brian Kung
http://about.me/briankung/
<http://www.facebook.com/brian.kung> <http://www.linkedin.com/in/briankung>
<http://www.twitter.com/briankung>

BE A UNICORN!